### PR TITLE
Create Color Checkbox

### LINKED ISSUE #8: Create ColorCheckbox Component
This component extends from `Checkbox` and it is specific because check icon is color dependent.

Current implementation:
Screenshot 2019-05-06 at 11 26 11

Design:
https://app.zeplin.io/project/58e24f0b8c7222d370f8c40c/screen/5cc463da9163353db1c37111

This component will have additional prop color (hex string) which will apply color to checkbox icon.

### PR DESCRIPTION
**Pull request by DelboyTrotter**
Fix #8

### CODE REVIEW

**Comment by manico** [comment]
Since this component in style is very much different, it is required to not inherit css class.
I removed checkbox class here, so styling should be fixed according to design.

**Comment by manico** [comment]
Please resolve conflicts locally.
Are design changes done?

**Comment by DelboyTrotter** [comment]
Yes, changes are done. I'm going to merge this today.

**Comment by dev387** [comment]
I will also need to use colors utility to create `ColorToggleButton`. When will it be merged to master?

File: src/utility/colors.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default {
+  hexToRgba(color, opacity) {
+    const hex = color.substring(1);
+    const r = parseInt(hex.substring(0, 2), 16);
+    const g = parseInt(hex.substring(2, 4), 16);
```
Comment (dev387) [inline src/utility/colors.js]: If we use 3digits hex code, this wont work. You should also add check for that.

**Pull request merged**